src/pages/Commits.tsx

```diff
@@ -99,41 +109,51 @@ export const Commits = () => {
   usePageTitle(`Project Health | ${projectId}`);
   useNetworkStatus(startPolling, stopPolling);
   const { mainlineCommits } = data || {};
-  const { versions } = mainlineCommits || {};
+  const { versions, nextPageOrderNumber, prevPageOrderNumber } =
+    mainlineCommits || {};
 
   const hasTaskFilter = filterTasks.length > 0;
-
   if (error) {
     return <PageDoesNotExist />;
   }
 
   return (
     <PageWrapper>
-      <HeaderWrapper>
-        <TupleSelectWrapper>
-          <TupleSelect options={tupleSelectOptions} />
-        </TupleSelectWrapper>
-        <StatusSelectWrapper>
-          <StatusSelect />
-        </StatusSelectWrapper>
-        <ProjectSelectWrapper>
-          <ProjectSelect selectedProject={projectId} />
-        </ProjectSelectWrapper>
-      </HeaderWrapper>
-      <BadgeWrapper>
-        <FilterBadges />
-      </BadgeWrapper>
-      <CommitsWrapper
-        versions={versions}
-        error={error}
-        isLoading={loading}
-        chartType={currentChartType}
-        hasTaskFilter={hasTaskFilter}
-      />
+      <PageContainer>
+        <HeaderWrapper>
+          <TupleSelectWrapper>
+            <TupleSelect options={tupleSelectOptions} />
+          </TupleSelectWrapper>
+          <StatusSelectWrapper>
+            <StatusSelect />
+          </StatusSelectWrapper>
+          <ProjectSelectWrapper>
+            <ProjectSelect selectedProject={projectId} />
+          </ProjectSelectWrapper>
+        </HeaderWrapper>
+        <BadgeWrapper>
+          <FilterBadges />
+        </BadgeWrapper>
+        <PaginationButtons
+          prevPageOrderNumber={prevPageOrderNumber}
+          nextPageOrderNumber={nextPageOrderNumber}
+        />
+        <CommitsWrapper
+          versions={versions}
+          error={error}
+          isLoading={loading}
+          chartType={currentChartType}
+          hasTaskFilter={hasTaskFilter}
+        />
+      </PageContainer>
     </PageWrapper>
   );
 };
 
+const PageContainer = styled.div`
```

Comment:
I feel like we include something like this on every page 😆 wonder if we should think about standardizing page containers at some point. but no need to address now!